projects/hello_adc/src/hello_adc.c

```diff
@@ -0,0 +1,39 @@
+#include "adc.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+
+#define BUTTON_PIN 178
+#define READ_FROM_PIN 166
```

Comment:
"Pin A6" and "Pin B2" doesn't refer to hex lol, the letter is the port (`GPIO_PORT_A`, `GPIO_PORT_B`, ...) and the number is the pin (0 through 15). So you don't need these defines; remove them and specify the addresses in the struct initializations in `main`.

(FYI: if in the future you need to write numbers which are best expressed in hex (which you don't need to for this homework!), use the `0x` prefix: `0x100 == 256`. `0b` also works for binary: `0b100 == 4`.)